### PR TITLE
-- added new kenyan local dialect hymnals ie (Kikuyu, dholuo and kisii)

### PR DESCRIPTION
**Pull request by TonyNais**
Hi Tinashe,

I took the liberty to add our local dialects to your awesome app, hope that's allowed. I also know you have a separate standard SDA hymnal app, but wouldn't it be great to have all the hymns 'under one roof'?

### CODE REVIEW

File: app/src/main/java/com/tinashe/hymnal/data/model/constants/Hymnals.kt [hunk 1]
```diff
@@ -20,7 +20,11 @@ enum class Hymnals(
     SWAHILI("swahili", "Nyimbo Za Kristo", "Swahili", R.raw.swahili),
     NDEBELE("ndebele", "UKrestu Esihlabelelweni", "Ndebele/IsiZulu", R.raw.ndebele),
     XHOSA("xhosa", "UKristu Engomeni", "IsiXhosa", R.raw.xhosa),
-    XITSONGA("xitsonga", "Risima Ra Vuyimbeleri", "Xitsonga", R.raw.xitsonga);
+    XITSONGA("xitsonga", "Risima Ra Vuyimbeleri", "Xitsonga", R.raw.xitsonga),
+    GIKUYU("gikuyu", "Nyimbo cia Agendi", "Kikuyu", R.raw.gikuyu),
+    ABAGUSII("abagusii", "Ogotera kw'ogotogia Nyasae", "Abagusii", R.raw.abagusii),
+    DHOLUO("dholuo", "Wende Nyasaye", "Dholuo", R.raw.dholuo),
+    ENGLISH1("english1", "SDA Hymnal", "English", R.raw.sdah);
```
Comment (TinasheMzondiwa) [inline app/src/main/java/com/tinashe/hymnal/data/model/constants/Hymnals.kt:27]: Can we please rename this to 

```SDAH("sdah", "SDA Hymnal", "English", R.raw.sdah);```

**Comment by TinasheMzondiwa** [comment]
Thank you @TonyNais for these changes.
Please just pull the latest `develop` into your branch and the ci should pass

**Comment by TonyNais** [comment]
Hey @TinasheMzondiwa 
I have renamed the English enum key.

**Review by TinasheMzondiwa — APPROVED** [review]
Thank you!

**Comment by Tadiwrr** [comment]
Hi Tinashe I love your app its amazing keep it up man, God bless you

**Comment by Tadiwrr** [comment]
I loved your update changed the ui drasticly and made it much easier to use

**Comment by Tadiwrr** [comment]
Wow all in Kotlin Amazing

**Pull request merged**